core/equipment.c

```diff
@@ -242,3 +242,26 @@ void reset_cylinders(struct dive *dive, bool track_gas)
 		cyl->deco_gas_used.mliter = 0;
 	}
 }
+
+#ifdef DEBUG_CYL
+void dump_cylinders(struct dive *dive, bool verbose)
+{
+	int i;
+
+	printf("Cylinder list:\n");
+	for (i = 0; i < MAX_CYLINDERS; i++) {
```

Comment:
if there are more variables in the function prologue or if the iterator `i` is re-used in a couple of loops it would be fine to add it on top, except in this case it's scope is only limited to the `for` loop, thus C99 allows us to do a:
```
for (int i = 0; i < MAX_CYLINDERS; i++) {
```